www/lobby/index.ts

```diff
@@ -41,5 +41,8 @@ lobby_router.post('/', async (req: Request, res: Response) => {
  * Body Params: id, refreshToken
  */
 lobby_router.get('/', async (req: Request, res: Response) => {
-  res.status(200).json('placeholder get');
+  const userId = req.body.id;
+  const user = await User.fromId(userId);
+  if (!user) return res.status(404).json('user not found in database').end();
+  res.status(200).json(user.lobbies);
```

Comment:
i would prefer if we returned an object.. maybe something like

```suggestion
  res.status(200).json({ lobbies: user.lobbies });
```